controllers/default_controller.py

```diff
@@ -21,22 +21,23 @@ def systemGetAll(host = None, user = None, password = None):
             units = []
             components = handle.query_children(in_dn="sys", class_id=x["ciscoXmlName"])
             for y in components:
-                subElement = {"relative_path": "/"+(vars(y))["dn"]}
+                subElement = {"relative_path": "/" + (vars(y))["dn"]}
                 units.append(subElement)
-            finalObjs[x["humanReadableName"]]=units
+            finalObjs[x["humanReadableName"]] = units
             handle.logout()
         return finalObjs
 
-def getRackmount(host = None, user = None, password = None):
+
+def getRackmount(host=None, user=None, password=None):
     data = []
     handle = UcsHandle(host, user, password, secure=False)
     if handle.login():
         computeRackUnit = handle.query_children(in_dn="sys", class_id="computeRackUnit")
         for x in computeRackUnit:
```

Comment:
This PR is only for the enabling of hound.  This issue will be addressed as part of [RAC-4150](https://rackhd.atlassian.net/browse/RAC-4150)